pkg/endpoint/policy.go

```diff
@@ -155,87 +155,143 @@ func getSecurityIdentities(owner Owner, selector *api.EndpointSelector) []policy
 	return identities
 }
 
-func (e *Endpoint) removeOldFilter(owner Owner, filter *policy.L4Filter) int {
+// removeOldFilter removes the old l4 filter from the endpoint and returns
+// a map of the L3-dependent L4 rules that were successfully removed if the
+// value is true, or in case of an error the value is false.
```

Comment:
Please specify what "if the value is true" means; the value of what?

Reply:
Of the key of the map, can you help here native english speaker? :)

Reply:
Fixed